generate_elasticsearch_files.py

```diff
@@ -4,7 +4,12 @@
 if __name__ == '__main__':
     parser = argparse.ArgumentParser()
     parser.add_argument('--ontology_dir', type=str)
-    parser.add_argument( '--availability_input_dir', type=str)
+    parser.add_argument('--availability_input_dir', type=str)
     parser.add_argument('--generate_availability', action='store_true')
+    parser.add_argument('--code_system_translations_folder', type=str, default='example/code_systems_translations')
```

Comment:
In alignment with other arg names use "dir" instead of "folder" to refer to some directory or folder